index.html

```diff
@@ -10,234 +10,252 @@
+              <span>CANOPY</span>
+              <span class="dot"></span>
 
-            <span>Back End Dev</span>
-            <span class="dot"></span>
+              <span>Back End Dev</span>
+              <span class="dot"></span>
 
-            <span>2015</span>
+              <span>2015</span>
+            </div>
+            <div class="description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </div>
+            <div class="button-links">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <a href="#" class="project">See Project</a>
           </div>
-          <div class="description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-          </div>
-          <a href="#" class="project">See Project</a>
+          <img class="img" src="image/card-4.svg" alt="" />
         </div>
       </div>
+    </div>
 
-      <div class="card">
-        <img class="img" src="image/card-4.svg" alt="">
-        <div class="content">
-          <div class="topic">Multi-Post <br /> Stories</div>
-          <div class="sub-topic">
-            <span>CANOPY</span>
-            <span class="dot"></span>
-
-            <span>Back End Dev</span>
-            <span class="dot"></span>
-
-            <span>2015</span>
-          </div>
-          <div class="description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
+    <div class="myself">
+      <div class="left">
+        <div class="description">
+          <h1 class="topic">About Myself</h1>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </div>
+        <div class="menuSection">
+          <a href="#" target="_blank">LET'S CONNECT</a>
+          <div class="socialMedia">
+            <img src="image/Twitter.png" alt="" />
+            <img src="image/Linkedin.png" alt="" />
+            <img src="image/Meduim.png" alt="" />
+            <img src="image/Github.png" alt="" />
+            <img src="image/angellist.png" alt="" />
           </div>
-          <a href="#" class="project">See Project</a>
         </div>
+        <div class="project">Get my resume</div>
```

Comment:
- [ ] Kindly according to the requirements add here all the button properties for the hover, pressed, and disabled as shown in Figma and because it is very good for user experience (UX)✅✅💯💯